app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# AccountActivationsController
+class AccountActivationsController < ApplicationController
+  before_action :load_user, :check_activation, only: :edit
+
+  def edit
+    if @user.activate
+      log_in(@user)
+      flash[:success] = t("users.mailer.activated")
+      redirect_to(@user)
+    else
+      flash[:danger] = t("users.mailer.invalid_activation_link")
+      redirect_to(root_path)
+    end
+  end
+
+  private
+
+  def load_user
+    @user = User.find_by email: params[:email]
+    return if @user
+
+    flash[:danger] = t("layouts.messages.user_not_found")
```

Comment:
```suggestion
    flash.now[:danger] = t("layouts.messages.user_not_found")
```
ri nha, render mà ko .now thì ko hiển thị mô